pkgs/by-name/su/supermariowar/package.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, enet
+, yaml-cpp
+, SDL2
+, SDL2_image
+, SDL2_mixer
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "supermariowar";
+  version = "2.0-unstable-2020-10-13";
+
+  src = fetchFromGitHub {
+    owner = "mmatyas";
+    repo = "supermariowar";
+    rev = "c9a383be729ea93c5d67333e86cae38cb68ceba1";
+    hash = "sha256-Ztu+f7oU0I3VWKaILh7LmMvn0O+5kpaW0wxPtcmfoGI=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    enet
+    yaml-cpp
+    SDL2
+    SDL2_image
+    SDL2_mixer
+    zlib
+  ];
+
+  cmakeFlags = [
+    "-DBUILD_STATIC_LIBS=OFF"
+    "-DSDL2MAIN_LIBRARY=${SDL2}/lib/libSDL2.so"
+    "-DXDG_APPS_INSTALL_DIR=share/applications"
+  ];
+
+  postPatch = ''
+    # Patching CMakeLists.txt files causing build errors
+    for d in smw leveleditor worldeditor; do
+      substituteInPlace src/$d/CMakeLists.txt \
+        --replace-fail "/usr/share/pixmaps/" "share/pixmaps/"
+    done
+  '';
+
+  postInstall = ''
+    mkdir -p $out/bin
+
+    for app in smw smw-leveledit smw-worldedit; do
+      chmod +x $out/games/$app
+
+      cat << EOF > $out/bin/$app
+      #!/bin/sh
+      $out/games/$app $out/share/games/smw
+    EOF
+      chmod +x $out/bin/$app
+    done
+  '';
+
+  meta = with lib; {
```

Comment:
Please remove `with lib;` and use `lib.` explicitly where needed, like in `lib.licenses.gpl2Plus`.

And format the whole file with `nixfmt-rfc-style` while you are at it.

Reply:
`with lib` removed and file formatted with the `nixfmt-rfc-style` nix package :+1:.